lib/serverspec/type/php_ext.rb

```diff
@@ -0,0 +1,10 @@
+module Serverspec::Type
+  class PhpExt < Base
+    def loaded?
+      ret = @runner.run_command("php --re '#{@name}'")
+      val = ret.stdout
```

Comment:
Just fyi, yeah, the error message pops up in stdout. And the return value is also 0, despite it being an error.